### PR TITLE
add/clean links/services

### PR DESCRIPTION
**Pull request by KazuCocoa**

### CODE REVIEW

**Comment by hotchemi** [comment]
👁

File: README.md [hunk 1]
```diff
@@ -52,13 +52,12 @@ Of course it sometimes includes google's official one.
 
 ## Recorder
 
-- [appium-dot-app](https://github.com/appium/appium-dot-app)
-- [Robotium Recorder](http://robotium.com/products/robotium-recorder)
```
Comment (KazuCocoa) [inline README.md:56]: the link was deprecated

File: README.md [hunk 2]
```diff
@@ -52,13 +52,12 @@ Of course it sometimes includes google's official one.
...
+- [appium-desktop](https://github.com/appium/appium-desktop)
+- [Espresso Test Recorder](https://developer.android.com/studio/test/espresso-test-recorder)
 
 ## Assertion
 
 - [AssertJ](http://joel-costigliola.github.io/assertj/)
-- [assertj-android](https://github.com/square/assertj-android)
```
Comment (KazuCocoa) [inline README.md:61]: deprecated

File: README.md [hunk 3]
```diff
@@ -84,18 +84,19 @@ Of course it sometimes includes google's official one.
...
-    - [TestObject](https://testobject.com/)
 - [Testdroid](http://testdroid.com/)
 - [Scirocco Cloud](http://www.scirocco-cloud.com/ja/price.html)
 - [Remote TestKit](https://appkitbox.com/)
-- [Xamarin Test Cloud](http://xamarin.com/test-cloud)
+- [App Center](https://appcenter.ms)
 - [Kobiton](https://kobiton.com)
 - [Mobilebox](http://mobileboxlab.com/en/)
+- [HeadSpin](https://www.headspin.io)
+- [test.ai](http://test.ai)
```
Comment (KazuCocoa) [inline README.md:101]: HeadSpin is kind of measuring performance service on real devices which exist in various countries. You can figure out performance issues **before** releasing.
test.ai is mobile-focused AI-driven test conducting service.

File: README.md [hunk 4]
```diff
@@ -84,18 +84,20 @@ Of course it sometimes includes google's official one.
...
+- [Magic Pod](https://magic-pod.com)
```
Comment (KazuCocoa) [inline README.md]: Magic Pod help users to implement test scenarios using AI power.

**Comment by KazuCocoa** [comment]
removed wip

**Comment by KazuCocoa** [comment]
will merge this later

**Comment by hotchemi** [comment]
👍

**Pull request merged**